cr17/nodes/cr17/high_level_state_controller.py

```diff
@@ -230,31 +235,47 @@ def run(self):
             if(self.autostate == "INIT"):
                 if(True):#change to some actual check
                     self.autostate = "F_OBSTACLE_FIELD"
+
+                    msg = Bool()
+                    msg.data = False
+                    self.puber.publish(msg)
+                    rospy.logwarn("Publishing out path desire.")
```

Comment:
Where is it publishing the out path desire? There should be no paths published in the if statements that are meant for state changes.